src/main/java/org/apache/sling/jcr/resource/internal/JcrValueMap.java

```diff
@@ -85,12 +85,19 @@ protected String checkKey(final String key) {
 
     /**
      * @see org.apache.sling.api.resource.ValueMap#get(java.lang.String, java.lang.Class)
+     * 
+     * Note: The {type} parameter is marked as @NonNull in the API documentation, but
+     * https://issues.apache.org/jira/browse/SLING-11567 it got obvious that this assumption
```

Comment:
invalid javadoc, rather `{code type}`

Reply:
The taglet format is `{@code }`: https://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#code

Reply:
my bad (first save, then commit)